doc/060_forget.rst

```diff
@@ -336,6 +343,50 @@ could specify:
 (Note that `1w` is not a recognized duration, so you will have to specify 
 `7d` instead)
 
+Security considerations in append-only mode
+===========================================
+
+To prevent a compromised backup client from deleting its backups (for example
+due to a ransomware infection), a repository service/backend can serve the
+repository in a so-called append-only mode. This means that the repository can
+only be written to and read from, while delete and overwrite operations are
+denied. Restic's `rest-server`_ features the append-only mode, but few other
+standard backends do. To support append-only with such a backend, one can use
+`rclone`_ as a complement in between the backup client and the backend service.
+
+.. _rest-server: https://github.com/restic/rest-server/
+.. _rclone: https://rclone.org/commands/rclone_serve_restic/
+
+To remove snapshots and recover the corresponding disk space, the ``forget``
```

Comment:
IMO this paragraph should also explain the append-only setup variant where a trusted host is allowed to remove data from the repository. Without such an explanation, the "trusted client" in the next paragraph is not properly defined.

Reply:
Isn't that already explained in the previous paragraph? I'm trying to come up with a short addition that makes it more clear what "trusted client" means but it all sounds redundant.

Perhaps we should instead change the phrasing of "a trusted client" into something like "a client with full access", especially since "trusted" is never used again (defining a term for single use is kind of silly). That makes the paragraph-length sentence even more convoluted though, so I've rewritten it a bit (in the upcoming commit).

Reply:
What about adding something like the following:
```
The usual setup is to use a separate, well-secured client that is full access to the repository and is only used to run ``prune``.
```

Reply:
I've incorporated your suggestion in the new text changes I just pushed, good input! I'll mark it as resolved now that @lgommans removed the reference to "trusted" earlier and I added your suggestion.